applications/NXcanSAS.nxdl.xml

```diff
@@ -278,7 +278,7 @@
         </doc>
       </attribute>
 
-      <field name="Q" type="NX_NUMBER" units="NX_PER_LENGTH">
+      <field name="Q" type="NX_NUMBER" units="NX_PER_LENGTH" nameType="specified">
```

Comment:
Note that this is not necessarily needed, because 'specified' is the default option for nameType.  (Only iff a name is not specified for a group nameType becomes by default 'any')

Reply:
Rather, since upper case has been the NeXus indicator that a name is flexible, let's leave this as a reminder, even though it is not needed, lest someone not associated with the canSAS think it to be flexible.